src/scripts/install.py

```diff
@@ -140,11 +144,20 @@ def parse_snowflake_url(url):
     if len(c) == 1:
         account = c[0]
     else:
+        # for customers
         if path.endswith("snowflakecomputing.com"):
             account = c[0]
             region = c[1] if len(c) == 4 else 'us-west-2'
+        else:
+        # for internal use
```

Comment:
as mentioned in our earlier meeting, please don't differentiate between internal and external use